css/styles.css

```diff
@@ -0,0 +1,527 @@
+    width: 100%;
+    z-index: 100;
+}
+  
+@media only screen and (max-width: 767px) {
+    .hidden-md-down {
+        display: none; 
+    } 
+}
+
+@media only screen and (min-width: 1400px) {
+    body {
+        max-width: 1400px;
+        margin: 0 auto;
+    }
+}
+
+
+/* TYPOGRAPHY */
+
+@font-face {
+    font-family: 'Anton';
+    src: url("../assets/fonts/Anton/Anton-Regular.ttf");
+}
+
+@font-face {
+    font-family: 'EB Garamond';
+    src: url("../assets/fonts/EB_Garamond/EBGaramond-Italic.ttf");
+}
+
+@font-face {
+    font-family: 'Montserrat';
+    src: url("../assets/fonts/Montserrat/Montserrat-Regular.ttf");
+}
+
+h1 {
+    font-family: 'Anton', sans-serif;
+    font-size: 7rem;
+    letter-spacing: 0.2rem;
+}
+
+h2 {
+    font-size: 3rem;
+    font-family: 'Anton', sans-serif;
+}
+
+h3 {
+    font-family: 'EB Garamond';
+    font-size: 3rem;
+}
+
+p, 
+h4 {
+    font-family: 'Montserrat', sans-serif;
+    font-size: 2rem;    
+}
+
+a {
+    text-decoration: none;
+    color: black; 
+}
+
+i {
+    font-size: 1.5rem; 
+}
+
+.dot {
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+
+
+/* TOP HEADER */
+header {
+    margin-bottom: 8vh;
+}
+
+header .top-header {
+    background: white;
+    padding: 10px 0 0 0;
+    width: 100%;
+}
+
+.top-header .brand {
+    text-align: center;
+}
+
+@media only screen and (min-width: 767px) {
+    .top-header .brand {
+        text-align: left;
+        text-align: initial;
+        padding: 0 15px;
+        width: 50%;
+        float: left;
+    }
+}
+
+.top-header .brand .name {
+    font-family: 'Anton', sans-serif;
+    letter-spacing: 0.2rem;
+    font-size: 1.3rem;
+}
+
+.top-header nav {
+    float: right;
+    padding: 0 15px;
+}
+
+.top-header nav ul li {
+    float: left;
+    position: relative;
+}
+
+.top-header nav ul li a {
+    padding: 0 10px; 
+}
+
+.top-header nav ul li a .fab {
+    font-size: 1.5rem;
+}
+
+
+/* 
+*
+*    SECTIONS 
+*
+*/
+
+
+/* HOWDY SECTION */
+.howdy {
+    background: #FC5956;
+    height: 100vh;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-align: center;
+        -ms-flex-align: center;
+            align-items: center;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+        -ms-flex-direction: column;
+            flex-direction: column;
+}
+
+.howdy h1 {
+    padding: 16vh 0; 
+    color: #f8f8f8;
+    font-size: 11rem;
+    margin: 0 auto;
+}
+
+.howdy .dot {
+    color: #1C1B1F; 
+}
+
+.howdy p {
+    font-size: 2.2rem;
+    color: #f8f8f8;
+    text-align: center;
+    margin: 0 auto;
+}
+
+.howdy strong {
+    color: #1C1B1F;
+}
+
+
+/* WORK SECTION */
+
+.work {
+    background: #f8f8f8;
+    padding-bottom: 8vh;
+}
+
+.work .work-header {
+    text-align: initial;
+    margin: 0 auto;
+    padding: 8vh 8vh 0 8vh;
+    font-size: 7rem;
+}
+
+.work .work-title p {
+    padding: 0 8vh 4vh 8vh;
+}
+
+.work .work-title .dot {
+    color: #FC5956
+}
+
+
+
```

Comment:
Inconsistent spacing between the end of a section and the start of the comment for the next one, line with comment "this one" for comparison.